test/unit/cache_bin.c

```diff
@@ -112,10 +112,10 @@ TEST_BEGIN(test_cache_bin) {
 		expect_true(cache_bin_ncached_get(&bin, &info)
 		    == ncached_max - i, "");
 		/*
-		 * This should fail -- the reduced version can't change low
-		 * water.
+		 * This should fail -- the easy variant can't change the low
+		 * water mark.
 		 */
-		ptr = cache_bin_alloc_easy_reduced(&bin, &success);
+		ptr = cache_bin_alloc_easy(&bin, &success);
 		expect_ptr_null(ptr, "");
 		expect_false(success, "");
```

Comment:
nit: these seem like something we missed to fill in.

Reply:
You mean the strings? I mostly don't add a message for things that are contextually clear. I can if you like, but I think I'd just be copying and pasting "expected failure" into the message for `expect_false(success)`. I'd just as soon refactor the test macros so that they don't have a required message parameter.

Reply:
Makes sense. I also often didn't like having to have the message, and relied on some macro to auto-fill it in.